x-pack/winlogbeat/module/pipeline.go

```diff
@@ -14,6 +14,6 @@ import (
 //go:embed */ingest/*.yml
 var pipelinesFS embed.FS
 
-func Init() {
+func init() {
```

Comment:
This changes the existing behaviour, does not it? AFAIK `Init` does not execute when the package is imported but `init` does. It could have been a bug or just dead code (unused public function).

Reply:
This depends on your concern about the scope of importers; the only importer of this package in beats is x-pack/winlogbeat/cmd which imports to call `Init()` in its only `init()` function. In that context this is not a change in behaviour. If we care about importers from outside then that's a different matter, but I don't think we do.